python/ray/job_config.py

```diff
@@ -30,14 +30,21 @@ def __init__(self,
         else:
             self.worker_env = worker_env
         if runtime_env:
+            # TODO(architkulkarni): move import to top of file (currently doing
+            # that causes pytest to raise ImportError for some reason)
```

Comment:
This sounds like a circular dependency? Are we importing anything from `JobConfig` inside of `runtime_support`?

Reply:
Wow, you're totally right.  Let me think about how to resolve this

Reply:
It turns out the circular dependency is unrelated, as removing the import from JobConfig gives the same error.  I found a related thread here https://stackoverflow.com/questions/41748464/pytest-cannot-import-module-while-python-can but the solutions there involving __init__.py didn't work.  It's taking some time to figure out, so I'll just leave it as an internal import, as it was before this PR.